service/broadcast/src/test/java/org/kaazing/gateway/service/broadcast/BroadcastServiceTest.java

```diff
@@ -355,20 +370,23 @@ private CountDownLatch getLatch() {
 
```

Comment:
The running member variable of FastTestBackendProducer (at line 369) should be volatile, otherwise the stop() method may not work (since is called from a thread other than the thread the producer is running on). That may explain why the test still hangs on Travis.